PyTorch/Segmentation/MaskRCNN/pytorch/maskrcnn_benchmark/config/paths_catalog.py

```diff
@@ -5,15 +5,15 @@
 
 
 class DatasetCatalog(object):
-    DATA_DIR = "/datasets"
+    DATA_DIR = "/shared/data"
```

Comment:
Is there a way to get the `DATA_DIR` from command line? because the actual data stored in `/scratch/data/mask_rcnn/pytorch`

Reply:
I can hardcode it there for now.
It's a config file.